src/details/ArborX_DetailsBatchedQueries.hpp

```diff
@@ -47,7 +48,19 @@ struct BatchedQueries
   // indirection when recording results rather than using that function at
   // the end.  We decided to keep reversePermutation around for now.
 
-  template <typename ExecutionSpace, typename Predicates,
+  template <typename BoundingVolume>
+  static auto convertToBox(BoundingVolume const &bounding_volume)
```

Comment:
Not sure what to call it or where to place it, but functionally it does the right thing.